apps/relying-party-func/host.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": "2.0",
+  "logging": {
+    "applicationInsights": {
+      "samplingSettings": {
+        "isEnabled": true,
+        "excludedTypes": "Request"
```

Comment:
why we excude requests? which percentage of sampling?

Reply:
@Garma00 we can remove `samplingSettings` for now

Reply:
[5de1324](https://github.com/pagopa/io-fims/pull/18/commits/5de1324e3e974d1db9eb5183250c4bc016b24860)